internal/providers/terraform/azure/testdata/cosmosdb_cassandra_keyspace_test/cosmosdb_cassandra_keyspace_test.golden

```diff
@@ -0,0 +1,44 @@
+
+ Name                                                                   Monthly Qty  Unit                      Monthly Cost 
+                                                                                                                            
+ azurerm_cosmosdb_cassandra_keyspace.autoscale                                                                              
+ ├─ Provisioned throughput (standard, West US)                                   60  RU/s x 100                     $350.40 
```

Comment:
I think this is still not correct because the cost is being calculated based on the max_throughput, not a usage parameter.